test/EFCore.PG.FunctionalTests/Query/NetworkQueryNpgsqlTest.cs

```diff
@@ -239,7 +240,7 @@ public void PhysicalAddress_macaddr_LessThan_macaddr()
         /// <summary>
         /// Tests translation for <see cref="NpgsqlNetworkExtensions.LessThan(DbFunctions,PhysicalAddress,PhysicalAddress)"/>.
         /// </summary>
-        [Fact]
+        [MinimumPostgresVersionFact(10, 0)]
```

Comment:
@austindrenski FYI I'm going to be removing all these in the preview7 PR, see https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/pull/943#discussion_r307014312. Version checks made sense before because refraining from translating would allow client evaluation, but in 3.0 that doesn't exist anymore. So I think it makes sense to simply translate and let PostgreSQL error.

We can still look at versions while translating in case some later version of PostgreSQL allows better translations - but that's not the case here.

Let me know if you think differently about this.

Reply:
Where these attributes are applied, there is no fallback (e.g. the function just doesn't exist in prior versions), so I'm not sure what the alternative would be...

Are you thinking about conditionally `Assert.Throws<T>`?

There were a few places where I modified tests to check whether the right SQL for the right version was emitted (e.g. custom sequences with non-`bigint` data types).

Reply:
Yeah looks like tests for 9.6 are failing in #943 due to `macaddr8` not existing.

Reply:
Please ignore my silly comment above - this is about skipping tests on old versions of PostgreSQL, which makes total sense.